docs/reference/lidar.md

```diff
@@ -547,7 +547,7 @@ from controller import Lidar
 
 class Lidar (Device):
     def getPointCloud(self, data_type='list'):
-    def getLayerPointCloud(self, layer):
+    def getLayerPointCloud(self, layer, data_type='list'):
```

Comment:
This is a new parameter, maybe we should merge it in the `develop` branch

Reply:
If it would be mandatory, then yes.
But it is optional, so it doesn't break the compatibility.

I can make a version without it in master and then add it in develop. But I'm not sure it is worth it.

Reply:
> But I'm not sure it is worth it.

I agree, too much hustle for a tiny impact :)